x/erc20/keeper/proposals.go

```diff
@@ -74,6 +89,7 @@ func (k Keeper) RegisterERC20(
 		)
 	}
 
+	// TODO: This should probably also convert the ERC20 to a precompile and change the owner ?
```

Comment:
can you elaborate?

Reply:
when creating a Coin from an ERC20 it's expected that the owner becomes `OWNER_MODULE` so that the correct `isNative` checks in the middleware are kept intact?

Reply:
Actually it should stay `External` because it will be important for the later to restrict it for native ERC20s.